node/src/tests.rs

```diff
@@ -321,17 +340,18 @@ impl IntegrationTestSetup {
                     near_signer_key: ed25519_dalek::SigningKey::generate(&mut OsRng),
                     near_responder_keys: vec![ed25519_dalek::SigningKey::generate(&mut OsRng)],
                 },
-                local_storage_aes_key: rand::random(),
+                local_storage_aes_key: rand::random(), // change this after the fact
```

Comment:
Not following this comment, can you elaborate?

Reply:
Not relevant anymore. Removed.